### PR TITLE
[Fix Screenshots] Artifact Hub under "How it Works See It in Action” section

### LINKED ISSUE #4850: [Screenshots] Artifact Hub under 'How it Works See It in Action' section
### Current Behavior
Currently, [every integration](https://layer5.io/cloud-native-management/meshery/integrations/artifact-hub) has only two old screenshots of the designer and visualizer in the 'How it Works See It in Action' section.

### Desired Behavior
- Add new images and videos from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap))
- Add screenshots for [Artifact Hub](https://layer5.io/cloud-native-management/meshery/integrations/artifact-hub) integration.

### Acceptance Tests
- Ensure your screenshot represents the use of a few specific integrations.
- The screenshot from the Designer mode should contain a valid design, refer to [meshery catalog](https://meshery.io/catalog) to use and view published designs.
- Add both Dark Mode and Light Mode screenshots.
- The screenshot should not be a snippet, take the full screenshot of your browser.
- Use Incognito mode on Chrome browser, to ensure your bookmark bar or any other items are not shown in the screenshot/video.

---

<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by sandramsc**
**Description**

This PR fixes #4850

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/integrations/_images/artifact-hub/meshmap-artifact-hub-visualizer-dark.png
Comment (leecalcote) [inline src/collections/integrations/_images/artifact-hub/meshmap-artifact-hub-visualizer-dark.png:1]: @aabidsofi19, that pattern icon color. Yikes.

**Comment by l5io** [comment]
🚀 Preview for commit 4584cae342d33702655dd1692cbeac2d4fec95b7 at: https://6568b2785fe521009479318d--layer5.netlify.app

**Comment by sandramsc** [comment]
Screenshots render fine, not sure why the  **lighthouseci / lighthouseci (pull_request)** was cancelled after 10sec..

**Comment by leecalcote** [comment]
> Screenshots render fine, not sure why the **lighthouseci / lighthouseci (pull_request)** was cancelled after 10sec..

Oh, good question. I canceled those given the high volume of builds that the site was doing earlier today. Those builds were redundant.

**Comment by l5io** [comment]
🚀 Preview for commit f295255442833260ed3c664b4e0a38aeb1fadf6d at: https://656d8c7319315a23781c9406--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit d2847c45c9c9482a4ecac949ebd5f35a14775cea at: https://657656da40c3bb01b89508d5--layer5.netlify.app

**Review by leecalcote — APPROVED** [review]

**Pull request merged**